api_volontaria/apps/user/views.py

```diff
@@ -51,8 +51,12 @@ class UserViewSet(viewsets.ModelViewSet):
     filter_fields = '__all__'
 
     def get_serializer_class(self):
-        if (self.action == 'update') | (self.action == 'partial_update'):
-            return serializers.UserUpdateSerializer
+        # TODO Commented section, the UserUpdateSerializer does not exist !
+        # if (self.action == 'update') | (self.action == 'partial_update'):
+        #     return serializers.UserUpdateSerializer
+
```

Comment:
Ouch! Good one!!

I will make an analysis but for the moment you can just remove this code since it doesn't work (we have no error 500 in prod and test are passing so it's not used at all.

We will add a new ticket to test this part and fill the hole.

Reply:
Okay perfect !